devel/jenkins/Portfile

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           java 1.0
+
+name                jenkins
+version             2.401.3
+revision            0
+
+homepage            https://jenkins.io
+
+description         The leading open source automation server,\
+                    Jenkins provides hundreds of plugins to support building,\
+                    deploying and automating any project.
+
+long_description    ${description}
+
+platforms           darwin
```

Comment:
this is the default and can thus be removed.

Reply:
removed